spec/support/wait_for_ajax.rb

```diff
@@ -1,15 +1,15 @@
-module WaitForAjax
-  def wait_for_ajax
-    Timeout.timeout(Capybara.default_wait_time) do
-      loop until finished_all_ajax_requests?
-    end
-  end
-
-  def finished_all_ajax_requests?
-    page.evaluate_script('jQuery.active').zero?
-  end
-end
-
-RSpec.configure do |config|
-  config.include WaitForAjax, type: :feature
-end
+module WaitForAjax
+  def wait_for_ajax
+    Timeout.timeout(Capybara.default_wait_time) do
+      loop until finished_all_ajax_requests?
+    end
+  end
+
+  def finished_all_ajax_requests?
+    page.evaluate_script('jQuery.active').zero?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.